libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -102,12 +110,15 @@
         [style.maxHeight]="!mobile ? maxHeight : 'auto'"
         (focusEscapeList)="handleListFocusEscape($event)"
         [style.minWidth]="'100%'"
+        role="listbox"
+        aria-roledescription="Suggestion Found"
```

Comment:
Please use @Input() for aria-roledescription="Multi Value Input" so the customers can pass own aria-roledescription value.